drivers/sensor/iis2iclx/iis2iclx_trigger.c

```diff
@@ -228,18 +221,18 @@ int iis2iclx_init_interrupt(const struct device *dev)
 	iis2iclx->work.handler = iis2iclx_work_cb;
 #endif /* CONFIG_IIS2ICLX_TRIGGER_OWN_THREAD */
 
-	ret = gpio_pin_configure(iis2iclx->gpio, cfg->irq_pin,
-				 GPIO_INPUT | cfg->irq_flags);
+	ret = gpio_pin_configure(cfg->gpio_drdy.port, cfg->gpio_drdy.pin,
```

Comment:
We could use `gpio_pin_configure_dt` here.

Reply:
Excellent!

Reply:
Yeah, neat. That's thanks to @mbolivar-nordic.